### PR TITLE
Add lastSeenDateTime property

### PR DESCRIPTION
**Pull request by jahsu-MSFT**

### CODE REVIEW

File: api-reference/beta/resources/printer.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ This resource supports:
 |location|[printerLocation](printerlocation.md)|The physical and/or organizational location of the printer.|
 |defaults|[printerDefaults](printerdefaults.md)|The printer's default print settings.|
 |capabilities|[printerCapabilities](printercapabilities.md)|The capabilities of the printer associated with this printer share.|
+|lastSeenDateTime|DateTimeOffset|The dateTimeOffset when a printer interacted with Universal Print. Read-only.|
```
Comment (nilakhan) [inline api-reference/beta/resources/printer.md:52]: The most recent dateTimeOffset when a printer interacted with Universal Print

File: changelog/Microsoft.PrintService.json [hunk 2]
```diff
@@ -14,7 +32,7 @@
       "Id": "118e092d-5c9b-4d50-928f-12aaa33abaf6",
       "Cloud": "prd",
       "Version": "beta",
-      "CreatedDateTime": "2020-01-27T00:00:00",
+      "CreatedDateTime": "2021-01-27T00:00:00",
```
Comment (nilakhan) [inline changelog/Microsoft.PrintService.json:35]: Why this is changed?

**Review by nilakhan — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.PrintService.json (hunk 2 above)
Comment (jahsu-MSFT) [reply changelog/Microsoft.PrintService.json:35]: The year for this change is actually wrong.

Comment (Lauragra) [reply changelog/Microsoft.PrintService.json:35]: The changelog entry will show the date on which the docs are published. When we publish this PR, we'll change the CreatedDateTime value to the current date.

Comment (Lauragra) [reply changelog/Microsoft.PrintService.json:35]: Never mind, this was a comment that wasn't about the entry associated with the actual PR.

**Review by Lauragra — APPROVED** [review]

**Comment by jahsu-MSFT** [comment]
@Lauragra Thanks for updating it!

**Pull request merged**